recipes/libmatio/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.5.6" %}
+package:
+  name: libmatio
+  version: {{ version }}
+
+source:
+  fn: matio-{{ version }}.tar.gz
+  url: https://github.com/tbeu/matio/releases/download/v{{ version }}/matio-{{ version }}.tar.gz
+  md5: bb53fd06f7ebb74589e9dd684b1b66a6
+
+build:
+  number: 0
+  skip: true     # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - gcc
+    - hdf5
+    - zlib
+
+  run:
+    - hdf5
+    - zlib
+
+test:
+  commands:
+    - test -e $PREFIX/include/matio.h  # [unix]
+
+about:
+  home: https://sourceforge.net/projects/matio
+  license: BSD-2-Clause
+  summary: matio is an C library for reading and writing MATLAB MAT files.
+
+extra:
+  recipe-maintainers:
+    - bioidiap
```

Comment:
How do we feel about having an org here, @ocefpaf @pelson? Personally, I think I would want to see individual people from that org so we aren't pinging the whole org. Plus, I'm not sure how our logic behaves there. Then again I could see some pluses to this. Thoughts?